openpower/configs/linux/skiroot_defconfig

```diff
@@ -3,6 +3,7 @@ CONFIG_ALTIVEC=y
 CONFIG_VSX=y
 CONFIG_SMP=y
 CONFIG_NR_CPUS=2048
+CONFIG_CPU_LITTLE_ENDIAN=y
```

Comment:
@jk-ozlabs - Is this why we are getting compile failures on any system that isn't upgraded to BE?  Do we need to make a separate defconfig for BE?  Or should we just deprecate it?  I know there are systems managed outside of github that will start to fail until they make the appropriate LE changes.